app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,73 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: null,
+  tags: [],
 };
 
+export const CONTACTS = [
+  {
+    label: "Oliver Smith",
+    value: "Oliver",
+  },
+  {
+    label: "Sam Smith",
+    value: "Sam",
+  },
+  {
+    label: "Nancy Binani",
+    value: "Nancy",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Getting started",
+    value: "value1",
+  },
+  {
+    label: "Onboarding",
+    value: "value2",
+  },
+  {
+    label: "UX",
+    value: "value3",
+  },
+  {
+    label: "Bugs",
+    value: "value4",
+  },
+  {
+    label: "User flow",
+    value: "value5",
+  },
+  {
+    label: "V2",
+    value: "value6",
+  },
+];
+
```

Comment:
Go through the Slack thread. We can build these values by creating a util called something like `buildSelectOptions`.